### DISCUSSION TITLE
Status Bug in SubQuery Indexer Admin

### DISCUSSION THREAD

**Discussion by rufatnas23** [subquery/subql#1003] [Frontier TestNet - Bug Reporting]
Sometimes it shows not indexing when i view my Projects, after restarting the page it shows Ready again.
![17f4cfd4-a7e0-4abd-add9-954b9a0bbafc](https://user-images.githubusercontent.com/94823027/167564720-2e4bf136-e144-4d4e-9723-a32e7112c1e4.png)
![e93c035c-26ae-438c-a58a-4c788c8c3b30](https://user-images.githubusercontent.com/94823027/167564729-86afd254-2ef0-4269-9328-ee92ce53293c.png)

**Comment by yasiryagi** [comment]
already reported https://github.com/subquery/subql/discussions/976

**Comment by mzxyz** [comment]
Resolved in `coordinator:v0.1.10`
